packages/xrpl/test/integration/transactions/payment.test.ts

```diff
@@ -32,4 +45,59 @@ describe('Payment', function () {
     },
     TIMEOUT,
   )
+
+  it(
+    'Validate Payment transaction v2 API: Payment Transaction: Specify Only Amount field',
+    async () => {
+      const result = await testTransaction(
+        testContext.client,
+        paymentTxExample,
+        paymentTxWallet,
+      )
+
+      assert.equal(result.result.engine_result_code, 0)
+      assert.equal((result.result.tx_json as Payment).Amount, AMOUNT)
+    },
+    TIMEOUT,
+  )
+
+  it(
+    'Validate Payment transaction v2 API: Payment Transaction: Specify Only DeliverMax field',
+    async () => {
+      const paymentTx = paymentTxExample
+      // @ts-expect-error -- DeliverMax is a non-protocol, RPC level field in Payment transactions
+      paymentTx.DeliverMax = paymentTx.Amount
+      // @ts-expect-error -- DeliverMax is a non-protocol, RPC level field in Payment transactions
+      delete paymentTx.Amount
+
+      const result = await testTransaction(
+        testContext.client,
+        paymentTx,
+        paymentTxWallet,
+      )
+
+      assert.equal(result.result.engine_result_code, 0)
+      assert.equal((result.result.tx_json as Payment).Amount, AMOUNT)
+    },
+    TIMEOUT,
+  )
+
+  it(
+    'Validate Payment transaction v2 API: Payment Transaction: identical DeliverMax and Amount fields',
+    async () => {
+      const paymentTx = paymentTxExample
```

Comment:
```suggestion
```
ditto and all other places

Reply:
updated in [28d1ef4](https://github.com/XRPLF/xrpl.js/pull/2689/commits/28d1ef47d738527dc4d4774c81629f0bae8dc521)